src/unregister.test.js

```diff
@@ -494,7 +494,46 @@ describe('unregisterNode()', () => {
     ])
   })
 
-  test('unregistering a pibling of the focused node', () => {
+  test('unregistering a pibling of the focused node should not recalculate focus', () => {
```

Comment:
```suggestion
  test('unregistering a sibling of the focused node should not recalculate focus', () => {
```

Reply:
I thought it was a spelling mistake as well, but it seems intentional since it refers to the parents sibling, not the actual sibling of the focused node. I'm happy to change to "parents sibling" if pibling isn't an actual word 😅

Reply:
Well, well, well... TIL! Let's leave it as a gem for future generations to discover 💎